docker/kubernetes/sawtooth-kubernetes-default.yaml

```diff
@@ -63,7 +63,7 @@ items:
               - -c
               - "sawadm keygen \
               && sawtooth keygen my_key \
```

Comment:
This key isn’t used anymore, after this change.

Reply:
This has been addressed.

Reply:
FWIW the root cause was not using the same key for genesis and the proposal.